docs/RelayAPI.md

```diff
@@ -1,376 +1,504 @@
-}
+### Trustline of user in currency network
+### `/networks/:network/users/:userA/trustlines/:userB`
+Returns a trustline between users A and B in a currency network.
+#### Request
 ```
-
-## Get transaction infos
-Returns the transaction information
-
-`GET /users/:userAddress/txinfos`
-
+GET /networks/:network/users/:userA/trustlines/:userB
+```
+#### Parameters
+|Name|Type|Required|Description|
+|-|-|-|-|
+|network|string|YES|Address of currency network|
+|userA|string|YES|Address of user A|
+|userB|string|YES|Address of user B|
 ### Response
 ```javascript
 {
-    "balance": "1000",
-    "nonce": 15,
-    "gasPrice": "10000"
+	"address": "0x7Ec3543702FA8F2C7b2bD84C034aAc36C263cA8b", // address of trustline counterparty / user B
+	"balance": -102, // balance of trustline from POV of user A
+	"given": 10000,  // credit line given to counterparty
+	"received": 10000, // credit line received by counterparty
+	"leftGiven": 10102, // given - balance
+	"leftReceived": 9898, // received - balance
+	"id": "0x314338891c370d4c77657386c676b6cd2e4862af1244820f9e7b9166d181057f" // identifier of trustline
 }
 ```
 
-## `NEW` Get balance of externally owned account
-Returns the ETH balance of an externally owned account
-
-`GET /users/:userAddress/balance`
+---
 
+### Total spendable amount of user in currency network
+### `/networks/:network/users/:user/spendable`
+Returns the total amount a user can spend in a currency network.
+#### Request
+```
+GET /networks/:network/users/:user/spendable
+```
+|Name|Type|Required|Description|
+|-|-|-|-|
+|network|string|YES|Address of currency network|
+|user|string|YES|Address of user|
 ### Response
 ```javascript
-"12.1009234"
+1000 // total amount user can spend in a currency network
 ```
 
-## Relay transaction
-Sends a signed transaction
-
-Remove txId as response instead compute on client
+---
 
-`POST /relay`
-
-### Request
-```javascript
-// rawTransaction
+### Spendable amount to other user in currency network
+### `/networks/:network/users/:userA/spendables/:userB`
+Returns amount user A can spend to user B in a currency network.
+#### Request
 ```
-
+GET /networks/:network/users/:userA/spendables/:userB
+```
+#### Parameters
+|Name|Type|Required|Description|
+|-|-|-|-|
+|network|string|YES|Address of currency network|
+|userA|string|YES|Address of user A|
+|userB|string|YES|Address of user B|
 ### Response
 ```javascript
-"txId"
+90 // amount user A can spend to user B in a currency network
 ```
 
-# Exchange API
-
-## `TODO` Get balances
-Returns balances of user
-
-`GET networks/:networkAddress/users/:userAddress/exchanges/:currency`
+---
 
+### Transfer path in currency network
+### `/networks/:network/path-info`
+Returns the cheapest path, the estimated gas costs and maximal fees for a transfer.
+#### Request
+```
+POST /networks/:network/path-info
+```
 ### Parameters
-- `currency` -
-
+|Name|Type|Required|Description|
+|-|-|-|-|
+|network|string|YES|Address of currency network|
+### POST body
```

Comment:
Those are the parameters. 
It should be started somewhere that they are posted as JSON. There should be an example.